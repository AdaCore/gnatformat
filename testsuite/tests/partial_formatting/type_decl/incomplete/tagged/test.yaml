description: IncompleteTaggedTypeDecl formatting for line selection.
driver: partial_gnatformat
args:
  [
    "test.ads",
    "-SL 74",
    "-EL 75",
  ]
status_code: 1