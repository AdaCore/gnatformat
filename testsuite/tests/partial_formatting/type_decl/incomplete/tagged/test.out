<CompilationUnit test.ads:27:1-149:23>
test.ads(<IncompleteTaggedTypeDecl ["Car"] test.ads:74:4-74:23>) - 74:1-74:23
^
   type Car is tagged;$

